src/plugins/layout/shadowdom.ts

```diff
@@ -69,59 +63,55 @@ export class ShadowDom {
 
     assert(!!parent, "moveShadowNode", "parent is missing");
```

Comment:
>assert [](start = 4, length = 6)

maybe out of scope for here, but should we turn off clarity once some threshold of assertions fail?